lib/pgBackRest/Common/Ini.pm

```diff
@@ -42,212 +43,325 @@ use constant INI_KEY_FORMAT                                         => 'backrest
 use constant INI_KEY_VERSION                                        => 'backrest-version';
     push @EXPORT, qw(INI_KEY_VERSION);
 
-use constant INI_COMMENT                                            => '[comment]';
```

Comment:
This was originally intended to embed comments into .info and .manifest files for anybody reading them manually.  I eventually decided it was wasted effort and the code was removed, but the ini structures persisted, partly because without good testing I was afraid to break something by removing them.